internal/pkg/show/config/plugins/jenkins-pipeline-kubernetes.yaml

```diff
@@ -0,0 +1,14 @@
+tools:
+# name of the tool
+- name: jenkins-pipeline-kubernetes
+  # id of the tool instance
+  instanceID: default
+  # format: name.instanceID; If specified, dtm will make sure the dependency is applied first before handling this tool.
+  dependsOn: [ ]
+  # options for the plugin
+  options:
+    # jenkinsUrl, format: hostname:port
+    jenkinsUrl: localhost:8080
+    jenkinsUser: admin
+    githubRepoUrl: https://github.com/xxx/xxx.git
```

Comment:
What if the `Jenkinsfile` is not in the root directory? You can leave a `todo` here, if it's a lot of work to support it.